sdk/tests/js/webgl-test-utils.js

```diff
@@ -661,6 +661,7 @@ var setupIndexedQuadWithOptions = function (gl, options) {
 
   if (options.colorLocation !== undefined) {
     var colors = new Float32Array(numVerts * 4);
+    poffset = 0;
```

Comment:
This change seems right. But how did it work before? Do you think this code path was just not used in any other tests?

Reply:
The `colorLocation` option is never used in the code base. I used it for a first iteration of one of the tests and found it broken. I switched to setting things manually so that option's gone back to being unused.